workers/data_refinery_workers/processors/smasher.py

```diff
@@ -226,7 +240,8 @@ def _smash(job_context: Dict) -> Dict:
                     logger.warning("Column repeated for smash job!",
                                    input_files=str(input_files),
                                    dataset_id=job_context["dataset"].id,
-                                   processor_job_id=job_context["job"].id
+                                   processor_job_id=job_context["job"].id,
+                                   column=column
```

Comment:
What does `column` look like when it is logged like this?

Reply:
Header